systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -118,6 +119,7 @@ function checkDirty(obj)
 
     function model = compile(model)
       w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
+      model.gurobi_present = checkDependency('gurobi');
```

Comment:
Sounds like a red herring.  Just moved memory and the other bug reappeared..?

- Russ


> On Apr 2, 2015, at 4:28 PM, Robin Deits <notifications@github.com> wrote:
> 
> In systems/plants/TimeSteppingRigidBodyManipulator.m:
> 
> > @@ -118,6 +119,7 @@ function checkDirty(obj)
> >  
> >      function model = compile(model)
> >        w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
> > +      model.gurobi_present = checkDependency('gurobi');
> With this commit, we get the l_init_called failed error on the Atlas computers every time we try to start up the controller (specifically, every time we try to enter constructQPDataPointerMex. Switching it to model.gurobi_present = true prevents that error from occurring.
> 
> —
> Reply to this email directly or view it on GitHub.
>